### ISSUE TITLE
[suggestion] TalkBack and support traits

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#401]
```
Hello, I'm Hyongsop Kim in Korea and TalkBack user.
Using this screen reader, I have one suggestion.
As you know, TalkBack doesn't have a trait function.
Trait is the attribute that let screen reader treats custom control as 
a special control.
For example, if a developer adds clickable object in text view in iOs, it is 
possible tolet 
VoiceOver recognize text view object as a button.
But TalkBack doesn't have this function.
I think trait is surely needed, because if developer use custom control and no 
attribute, TalkBack 
user is a little confusing whether the object is clickable or not.
So by making trait, we can encourage developers can use these traits.
Please consider this issue.
Thank you.


```

Original issue reported on code.google.com by `khsruru@gmail.com` on 15 Oct 2014 at 2:33
